Packages/com.unity.inputsystem/InputSystem/Plugins/Linux/SDLDeviceBuilder.cs

```diff
@@ -0,0 +1,329 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine.Experimental.Input.LowLevel;
+using UnityEngine.Experimental.Input.Utilities;
+using System.Text;
+using UnityEngine.Experimental.Input.Layouts;
+
+namespace UnityEngine.Experimental.Input.Plugins.Linux
+{
+    [Serializable]
```

Comment:
Oh, that makes me realize that this requirement is actually no longer necessary as layout information is no longer persisted but rather re-registered from scratch on every domain reload. Something to fix on the input system side.